### DISCUSSION TITLE
Important: audio mixer managed by scenes

### DISCUSSION THREAD

**Discussion by GiovanniG11** [obsproject/obs-studio#5039] [Development Questions]
Hi, you've done amazing work for video, but this program can't be complete without the chance to edit audio settings for each scenes. At the moment the option is global, I'm using rtsp streams with audio and I can't add audio track separately. I just need to select which microphone use on different scenes but any setting affect other scenes too.
Thanks for understanding

**Comment by Fenrirthviti** [comment]
Please use our Ideas page for suggestions, and the support forums or Discord for assistance with available workarounds. This particular suggestion is already made here: https://ideas.obsproject.com/posts/232/let-us-lock-source-audio-in-each-scene-independently and also contains some viable workarounds in the meantime.

**Reply by GiovanniG11** [reply]
thanks for your kind answer and sorry for posting here. Seems there aren't workaround for me, since I?m using VLC streams, I have to double all sources to use them indipendently on scenes. Unlogical work, I hope OBS will introduce soon the audio management for scenes.
